### DISCUSSION TITLE
Multi selection of nodes

### DISCUSSION THREAD

**Discussion by Raketten1963** [vakata/jstree#2706] [Q&A]
Hi all,

I hope for your help again.....

I have a tree like below and I want the admin to setup for each user which nodes he has access to. Some user have access to all, some less. 

My first attempt was simply to use the selection status (no checkboxes like below), and this is doable, with the drawback, that you have to hope that the user holds down control key while toggling nodes on/off. One single mistake and you are back to a single node selected. I did not like this as users tend not to know the use of control key.

I then turned to checkbox, but that is causing me other troubles. I cannot select 'First group' without having at least one child under this also selected. The minute I deselect last child, 'First group' is automatically deslected. I tried cascading up and down, but no combination will allow user this desire.

Please tell me that I overlooked something, or let me have your opinions on how to proceed...

![image](https://user-images.githubusercontent.com/62338118/220835150-3c9d340b-5be5-46c2-bb2a-65df241a4b1b.png)

**Comment by vakata** [comment]
Try this setting:
https://www.jstree.com/api/#/?q=three&f=$.jstree.defaults.checkbox.three_state

Set it to false in your config.
